packages/contracts/contracts/interfaces/IUnirep.sol

```diff
@@ -69,6 +71,7 @@ interface IUnirep {
     error InvalidEpoch(uint256 epoch);
 
     error InvalidProof();
+    error InvalidHistoryTreeRoot(uint256 historyTreeRoot);
     error InvalidStateTreeRoot(uint256 stateTreeRoot);
     error InvalidEpochTreeRoot(uint256 epochTreeRoot);
```

Comment:
The 
```solidity
error InvalidEpochTreeRoot(uint256 epochTreeRoot);
```
can be removed. and remember to update documentation (error codes)
using `yarn contracts errorcode` to get all errors and error codes